app/core.py

```diff
@@ -0,0 +1,47 @@
+"""Main functionalities."""
+
+import app.image.image as image
+import app.image.color_format as color_format
+import app.parser.factory as factory
+
+
+def run_core_functionality(args):
+
+    image = create_Image(args)
+    parser = factory.ParserFactory.create_object(
+        determine_color_format(args["color_format"]))
+
+    if args["parse"] == True:
+        image = parser.parse(image.data_buffer,
+                             determine_color_format(args["color_format"]),
+                             args["resolution"][0], args["resolution"][1])
+        if args["display"] == True:
+            displayable_data = parser.get_displayable(image)
+
+
+def create_Image(args):
+
+    raw_data = image.RawDataContainer(None).from_file(args["FILE_PATH"])
```

Comment:
You are creating empty RawDataContainer object and then using classmethod that creates new object.
Simply use `image.RawDataContainer.from_file()`